app/src/main/java/com/seekerclaw/app/bridge/NodeControlClient.kt

```
@@ -110,20 +111,68 @@ object NodeControlClient {
+     *              (`pendingPersist:true`) → the caller must fail-closed;
+     *  - `false` = Node reached; nothing stranded (even if the summary flush
+     *              itself failed) → do NOT force a re-pair;
+     *  - `null`  = flush did not reach/parse Node (connect-refused, 401, timeout,
+     *              malformed body) → the caller fail-closes on the unknown.
+     */
+    suspend fun flushShutdown(): Boolean? = withContext(Dispatchers.IO) {
+        val body = postForBody("/shutdown/flush", "{}") ?: return@withContext null
+        try {
+            val json = JSONObject(body)
+            if (json.has("pendingPersist")) json.getBoolean("pendingPersist") else null
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)